src/main/java/com/aws/greengrass/shadowmanager/model/ShadowRequest.java

```diff
@@ -21,11 +21,12 @@
 public class ShadowRequest {
     String thingName;
     String shadowName;
+    String operation;
```

Comment:
should this be an enum to avoid the .toLowerCase()? The constructor could still take a string

Reply:
We can make it into an enum. Might look cleaner. I just thought for one `toLowerCase` function, do we need to add an enum?